### ISSUE TITLE
Notify Relay Operator if Relay is Down [Feature Request]

### ISSUE THREAD

**Issue by alltheseas** [sandwichfarm/nostr-watch#394]
**Is your feature request related to a problem? Please describe.**
As relay operator, I may not have an easy to way to be notified when my relay goes down.

**Describe the solution you'd like**
## User Story ##
As a relay operator who is managing my relay in a competitive environment and wants to minimize downtime, I want to receive immediate notifications when my relay goes down, so that I can investigate, take action, and restore my relay.

### Acceptance Criteria ###
1. Relay operator can opt in to receive notification if nostr.watch detects relay is down
2. If relay operator opts in, they must provide their npub
3. Notification sent via DM to relay operator npub over nostr

**Describe alternatives you've considered**

**Additional context**
-~is this an API call/a webhook?~
-~is it an email, or npub note?~ npub DM
-how does relay operator opt in
-future: would it make sense to charge for this service? how might nostr.watch charge for this service?

**Comment by dskvr** [comment]
A private message on nostr makes sense.

**Comment by dskvr** [comment]
There could be a free version that will notify less frequently, and a paid option, that notifies within a minute or so.

**Comment by dskvr** [comment]
Additionally, paid version could have other notification options:

1. High ping notification
2. NIP-11 modification notification
3. `payments_url` 404 notification
etc..

**Comment by dskvr** [comment]
Yes, there could be server configuration changes, DNS changes, etc, but there are already services that do this, and most sysadmins will have server and network monitoring tools. 

There could also be spikes in events, either positive or negative. Presently not tracking number of events/users on relays, but that will change.

**Comment by alltheseas** [comment]
👀
